workers/datasets_based/src/datasets_based/workers/first_rows.py

```diff
@@ -427,6 +446,8 @@ def compute_first_rows_response(
           If the split rows could not be obtained using the datasets library in normal mode.
         - [`~workers.first_rows.RowsPostProcessingError`]
           If the post-processing of the split rows failed, e.g. while saving the images or audio files to the assets.
+        - [`~workers.first_rows.TooManyColumnsError`]
+          If the number of columns (features) exceeds the maximum supported number of columns.
```

Comment:
TooBigContentError is missing in the docstring

Reply:
Done, I added it, thanks!